ouroboros-consensus/src/Ouroboros/Storage/FS/Handle.hs

```diff
@@ -0,0 +1,61 @@
+{-# LANGUAGE LambdaCase #-}
+
+module Ouroboros.Storage.FS.Handle where
```

Comment:
Let's add a doc string here to explain that this module is intended as support for implementing `HasFS` instances, and will not be used directly by client code.